app/views/state_file/questions/az_review/_az_retirement_income_deductions_review_header.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="white-group" id="retirement-income-subtractions">
+  <div class="spacing-below-25">
+    <h2 class="text--body text--bold spacing-below-5"><%=t(".title") %></h2>
+    <p><%=t(".subtitle") %></p>
+  </div>
+
+  <div class="spacing-below-5">
+    <ul class="list--bulleted">
+      <% current_intake.state_file1099_rs.each do |ten_ninety_nine_r| %>
```

Comment:
can we name this `state_file1099_r` here? I feel like it's easier to read and consistent with other parts of the code